OSDOCS-8983: Update Ushift docs to reflect lvmd.yaml not required (#69505)
openshift/openshift-docs: modules/microshift-lvms-system-requirements.adoc

[id="lvms-volume-group-name_{context}"]
== Volume group name

> If the LVMS configuration is not present at `/etc/microshift/lvmd.yaml`, MicroShift will attempt to assign a default volume group dynamically using the `vgs` command.

Reviewer: ```suggestion
If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command.
```
What are the conditions when it can and cannot auto-assign a VG? We should say. We are telling users here what the system requirements are to use LVMS. We don't want it disabled.

If there are no volume groups on the {microshift-short} host, LVMS is disabled.

If there is only one VG on the {microshift-short} host, that VG is used. If there are multiple volume groups, the group `microshift` is used. If the `microshift` group is not found, LVMS is disabled.

If you want to use a specific VG, LVMS must be configured to select that VG. You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.

You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.

> After MicroShift has started, the lvmd.yaml may be updated to include or remove VGs.  For the changes to take effect, restart the MicroShift service. If the `lvmd.yaml` is deleted, LVMS will then be disabled.

Reviewer: ```suggestion
After the {microshift-short} starts, the `lvmd.yaml` can be updated to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, LVMS is disabled.
```
"If the `lvmd.yaml` is deleted, LVMS is disabled." Is that true, or does MicroShift try to auto-assign a VG when it restarts (as we said in the beginning)?

If we don't have a VG with enough capacity for workload storage, do we still get the error? Remember that this section of the docs is "Using LVMS in MicroShift requires the following system specifications." We're telling users how to use LVMS, we are *not* describing the default state of the system. If an error occurs when the storage isn't configured correctly, we should still list that somewhere.

[id="lvms-volume-group-name_{context}"]
== Volume group name

> If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command. {microshift-short} will assign a default VG when

Reviewer: ```suggestion
If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command. {microshift-short} assigns a default VG when only one VG is found. If more than one VG is present, the VG named `microshift` is assigned as the default. If a VG named `microshift` does not exist, LVMS is not deployed.
```

> one VG is found.  If there are more than on VGs, {microshift-short} will assign the VG named `microshift` if it exists. If the `microshift` VG does not exist, LVMS is not deployed.

Reviewer 2: I think there is a typo here. It should be `one` instead of `on`

> one VG is found.  If there are more than on VGs, {microshift-short} will assign the VG named `microshift` if it exists. If the `microshift` VG does not exist, LVMS is not deployed.

Reviewer 2: I think the `on` should be a `one` in this sentence.

> one VG is found.  If there are more than on VGs, {microshift-short} will assign the VG named `microshift` if it exists. If the `microshift` VG does not exist, LVMS is not deployed.

Reviewer: ```suggestion

```

* If more than one VG is present, the VG named `microshift` is assigned as the default. 
* If a VG named `microshift` does not exist, LVMS is not deployed.

If there are no volume groups on the {microshift-short} host, LVMS is disabled.

If you want to use a specific VG, LVMS must be configured to select that VG. You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.

You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.

> After the {microshift-short} starts, you can update the `lvmd.yaml` to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, {microshift-short} attempts to find a default VG again.

Reviewer 3: Tiny nit, otherwise this also LGTM
```suggestion
After {microshift-short} starts, you can update the `lvmd.yaml` to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, {microshift-short} attempts to find a default VG again.
```

> After the {microshift-short} starts, the `lvmd.yaml` can be updated to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, {microshift-short} will attempt to find a default VG again.

Reviewer: ```suggestion
After the {microshift-short} starts, you can update the `lvmd.yaml` to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, {microshift-short} attempts to find a default VG again.
```
Docs style is all present-tense, no future.
